x/sequencer/types/bond.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+import (
+	commontypes "github.com/dymensionxyz/dymension/v3/x/common/types"
+	"github.com/dymensionxyz/sdk-utils/utils/ucoin"
+)
+
+var (
+	BondDenom = commontypes.DYMCoin.Denom
+
+	// for tests, real value is supplied by rollapp keeper
```

Comment:
why u need it in the `types` package if it's for tests?
put it in some test file

Reply:
because it's used by apptesting